lib/crazy_eight_game.rb

```diff
@@ -0,0 +1,161 @@
+require 'JSON'
+require 'rest-client'
+require 'pry'
+
+class CrazyEightGame < ActiveRecord::Base
+
+    def new_deck
+        self.deck_id = JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['deck_id']
+        update_remaining(JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['remaining'])
```

Comment:
```suggestion
        update_remaining(deck_json['remaining'])
```
Since you stored the value above, you don't have to hit the API a second time. In general GET's are expensive, they take time and resources from both you and the server. It's a good idea to minimize their usage if possible.